chain-signatures/contract/tests/tests.rs

```diff
@@ -371,3 +396,144 @@ async fn test_contract_initialization() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_contract_propose_update() {
+    let (_, contract, accounts, _) = init_env().await;
+    dbg!(contract.id());
+
+    test_propose_update_config(&contract, &accounts).await;
+    test_propose_update_contract(&contract, &accounts).await;
+}
+
+async fn test_propose_update_config(contract: &Contract, accounts: &[Account]) {
+    // contract should not be able to propose updates unless its apart of the participant/voter set.
+    let execution = contract
+        .call("propose_update")
+        .args_json(serde_json::json!({
+            "contract": vec![1, 2, 3],
```

Comment:
The options are `code` or `config`.

Reply:
thanks for noticing this one. I made name changes on the function